ios/src/TGMapViewController.mm

```diff
@@ -21,6 +21,131 @@ @interface TGMapViewController ()
 
 @implementation TGMapViewController
 
+- (void)setPosition:(TangramGeoPoint)position {
+    if (self.map) {
+        self.map->setPosition(position.longitude, position.latitude);
+    }
+}
+
+- (void)setPositionEasedAtPosition:(TangramGeoPoint)position withDuration:(float)duration {
+    [self setPositionEasedAtPosition:position withDuration:duration withEaseType:TangramEaseTypeQuint];
```

Comment:
I think we decided to make the default ease type "cubic" for consistency with the Android SDK.